bundles/org.openhab.binding.mqtt.homeassistant/src/main/resources/OH-INF/thing/homeassistant-channels.xml

```diff
@@ -56,7 +56,7 @@
 		<config-description-ref uri="channel-type:mqtt:ha-channel"/>
 	</channel-type>
 
-	<channel-type id="ha-gps-accuracy" advanced="true">
+	<channel-type id="ha-gps-accuracy">
```

Comment:
So there is an advanced and a non-advanced version of this channel? Or are you leaving this here to not break setups?